### PR TITLE
building landing fixes, sidebar changes

### PR DESCRIPTION
**Pull request by WRRicht3r**
- Removed repeated text from build
- Removed XST section in ecosystem
- Changed text in sidebar

### CODE REVIEW

File: src/.#sora-builders.md [hunk 1]
```diff
@@ -0,0 +1 @@
+williamrichter@vzug-as2tvi.home.1336
```
Comment (bgodlin) [inline src/.#sora-builders.md]: Was it added intentionally?

**Review by bgodlin — APPROVED** [review]

File: src/build.md [hunk 2]
```diff
@@ -1,68 +1,13 @@
...
-and all the submissions, as well as RFP and merged implementations,
-are publicly accessible in the **[SORA RFP Github
-Repository](https://github.com/sora-xor/rfps)** which ensures that
-full transparency and decentralized practices are always upheld.
-
-For more information, read the articles [_How to RFP_](rfp.md), and [_Social Insurance for Systematically Important Infrastructure_](social-insurance.md). To stay in the loop with the latest implementations of the SORA network and Polkaswap, make sure to follow the Ecosystem updates, and finally, to get a taste of what is to come, watch the **[SORA Development Public Request for Proposal (RFP) Process Github Repo](https://github.com/sora-xor/rfps)**!
+- The [SORA Builders Programme](/sora-builders.md): A developer programme aimed at onboarding and incubating projects looking to build on the SORA network
+- [Social Insurance for SII](/social-insurance.md) an insurance provided by the SORA community, through SORA governance for systemically important infrastructure
+- [Pallets](/pallets.md) unlike EVM that uses smart contracts, Substrate-based networks use pallets.
```
Comment (outoftardis) [inline src/build.md]:
```suggestion
- [Pallets](/pallets.md): While EVM uses smart contracts, Substrate-based networks use pallets
```

File: src/build.md [hunk 3]
```diff
@@ -1,68 +1,13 @@
...
-and the **SORA Builders Programme**. Progress from both these
-initiatives will be reported in the fortnightly **Ecosystem Updates**
-and all the submissions, as well as RFP and merged implementations,
-are publicly accessible in the **[SORA RFP Github
-Repository](https://github.com/sora-xor/rfps)** which ensures that
-full transparency and decentralized practices are always upheld.
-
-For more information, read the articles [_How to RFP_](rfp.md), and [_Social Insurance for Systematically Important Infrastructure_](social-insurance.md). To stay in the loop with the latest implementations of the SORA network and Polkaswap, make sure to follow the Ecosystem updates, and finally, to get a taste of what is to come, watch the **[SORA Development Public Request for Proposal (RFP) Process Github Repo](https://github.com/sora-xor/rfps)**!
+- The [SORA Builders Programme](/sora-builders.md): A developer programme aimed at onboarding and incubating projects looking to build on the SORA network
+- [Social Insurance for SII](/social-insurance.md) an insurance provided by the SORA community, through SORA governance for systemically important infrastructure
```
Comment (outoftardis) [inline src/build.md]:
```suggestion
- The [SORA Builders Programme](/sora-builders.md): A developer programme aimed at onboarding and incubating projects looking to build on the SORA network
- [Social Insurance for SII](/social-insurance.md): An insurance provided by the SORA community, through SORA governance for systemically important infrastructure
```

File: src/build.md [hunk 4]
```diff
@@ -1,68 +1,13 @@
...
+- [Pallets](/pallets.md) unlike EVM that uses smart contracts, Substrate-based networks use pallets.
+- [Technical stack](/technical-stack.md) built on Parity Substrate, the SORA network can be both a standalone chain, as well as Polkadot or Kusama parachains
+- [Consensus](/consensus.md) the algorithm used to validate blocks on SORA is called Nominated Proof of Stake, learn more about it in this section.
```
Comment (outoftardis) [inline src/build.md]:
```suggestion
- [Consensus](/consensus.md): Nominated Proof of Stake is the algorithm used to validate blocks on SORA
```

File: src/build.md [hunk 5]
```diff
@@ -1,68 +1,13 @@
...
-Repository](https://github.com/sora-xor/rfps)** which ensures that
-full transparency and decentralized practices are always upheld.
-
-For more information, read the articles [_How to RFP_](rfp.md), and [_Social Insurance for Systematically Important Infrastructure_](social-insurance.md). To stay in the loop with the latest implementations of the SORA network and Polkaswap, make sure to follow the Ecosystem updates, and finally, to get a taste of what is to come, watch the **[SORA Development Public Request for Proposal (RFP) Process Github Repo](https://github.com/sora-xor/rfps)**!
+- The [SORA Builders Programme](/sora-builders.md): A developer programme aimed at onboarding and incubating projects looking to build on the SORA network
+- [Social Insurance for SII](/social-insurance.md) an insurance provided by the SORA community, through SORA governance for systemically important infrastructure
+- [Pallets](/pallets.md) unlike EVM that uses smart contracts, Substrate-based networks use pallets.
+- [Technical stack](/technical-stack.md) built on Parity Substrate, the SORA network can be both a standalone chain, as well as Polkadot or Kusama parachains
```
Comment (outoftardis) [inline src/build.md]:
```suggestion
- [Technical stack](/technical-stack.md): Built on Parity Substrate, the SORA network can be both a standalone chain, as well as Polkadot or Kusama parachains
```

File: src/.#sora-builders.md (hunk 1 above)
Comment (WRRicht3r) [reply src/.#sora-builders.md]: whaat! where is that? and why is my computer connected via the dishwasher?

**Review by outoftardis — APPROVED** [review]

**Pull request merged**